kustomize/move2kube/base/03-knative-resources.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: error-trigger-move2kube
+spec:
+  broker: __broker_name__
+  filter:
+    attributes:
+      type: error
+  subscriber:
+
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: __workflow_name__
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: transformation-saved-trigger-move2kube
+spec:
+  broker: __broker_name__
+  filter:
+    attributes:
+      type: transformation_saved
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: __workflow_name__
+---
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: __broker_name__
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: __kfunction_name__
+spec:
+  template:
+    # Note: can't be easily replaced with kustomize but it's not strictly needed
+    # metadata:
+      # name: __kfunction_name__-v1
+    spec:
+      initContainers:
+        - name: volume-mount-hack
+          image: quay.io/quay/busybox:latest 
+          securityContext:
+            runAsUser: 0
+            allowPrivilegeEscalation: true
+          command: [ "sh", "-c", "cp /tmp/.ssh/id_rsa /etc/pre-install/. && chown 185 /etc/pre-install/id_rsa" ]
+          volumeMounts:
+            - name: ssh-priv-key
+              mountPath: "/tmp/.ssh/id_rsa"
+              subPath: id_rsa
+              readOnly: true
+            - name: pre-install
+              mountPath: /etc/pre-install
+      containers:
+        - image: serverless-workflow-m2k-kfunc
+          imagePullPolicy: Always
+          envFrom:
+          - configMapRef:
+              name: move2kube-config # defines BROKER_URL and MOVE2KUBE_API         
```

Comment:
I am a little confused still with this naming; I was looking for a specific CM for the kfunc but it makes sense to have only 1 CM shared across resources for a given workflow and I do not have a better idea :)